Automattic/wp-calypso
# New Page Layout Picker: new design with category names on the left and thumbnails on the right

#### Changes proposed in this Pull Request

Redesign of the new page modal that places categories on the left and layout previews on the right. See design mockups: p9Jlb4-27F-p2.

**Before**
<img width="1447" alt="Screenshot 2021-03-04 at 10 52 38 AM" src="https://user-images.githubusercontent.com/1500769/109877327-c2074000-7cd7-11eb-970e-02bb05191eb8.png">


**After**
<img width="1447" alt="Screenshot 2021-03-04 at 10 50 34 AM" src="https://user-images.githubusercontent.com/1500769/109877131-82d8ef00-7cd7-11eb-8320-ddc3db9bd656.png">


#### Testing instructions



* `install-plugin.sh etk update/page-layout-picker-design` on sandbox
* Create a new page on your sandboxed site, the page layout picker should appear
* Test it works, you can find a checklist of things to test in #49587



Related to #49587
